messages/src/protocols/out_of_band/invitation.rs

```diff
@@ -23,7 +23,11 @@ pub struct OutOfBandInvitation {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub handshake_protocols: Option<Vec<MessageType>>, // TODO: Make a separate type
     pub services: Vec<ServiceOob>,
-    #[serde(rename = "requests~attach")]
+    #[serde(
+        default = "Attachments::new",
+        skip_serializing_if = "Attachments::is_empty",
```

Comment:
Should it be solved by rather by wrapping `Option` around `requests_attach`? Cause currently OOB sender will produce correct OOB if no attachments are present, but OOB receiver won't be able to parse it, since `requests_attach` is required.
And I suppose we are missing test-case for that